rust/ql/lib/codeql/rust/elements/internal/OperationImpl.qll

```diff
@@ -0,0 +1,29 @@
+/**
+ * Provides classes for operations.
+ *
+ * INTERNAL: Do not use.
+ */
+
+private import rust
+private import codeql.rust.elements.internal.ExprImpl::Impl as ExprImpl
+
+/**
+ * INTERNAL: This module contains the customizable definition of `Operation` and should not
+ * be referenced directly.
+ */
+module OperationImpl {
```

Comment:
nit: To follow the pattern for all the other `Impl` files, simply name this module `Impl`.

Reply:
Done.  I should get a bit more familiar with these patterns.